src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+---
+title: JavaScript Eventing Deep Dive
+subhead:
+  '`preventDefault` and `stopPropagation`: when to use which and what exactly each method does.'
```

Comment:
You don't have an intro, so maybe make this a tiny bit longer?

'The Event methods `preventDefault` ...

Reply:
I've updated the text to have a brief into. Let me know what you think?